src/style.css

```diff
@@ -0,0 +1,104 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
```

Comment:
this font should have the same name – _Avenir_, just change its font-weight so that it can be used the following way
```
.class1 {
  font-family: Avenir, ...;
  font-weight: 300;
}

.class2 {
  font-family: Avenir, ...;
  font-weight: 900;
}
```